p2p/net/network.go

```diff
@@ -6,21 +6,21 @@ import (
 	"github.com/spacemeshos/go-spacemesh/common/types"
 	"github.com/spacemeshos/go-spacemesh/log"
 	"github.com/spacemeshos/go-spacemesh/p2p/config"
-	"github.com/spacemeshos/go-spacemesh/p2p/delimited"
 	"github.com/spacemeshos/go-spacemesh/p2p/node"
 	"github.com/spacemeshos/go-spacemesh/p2p/p2pcrypto"
 	"github.com/spacemeshos/go-spacemesh/p2p/version"
 	"net"
 	"strconv"
 	"sync"
+	//"syscall"
 	"time"
 )
 
 // DefaultQueueCount is the default number of messages queue we hold. messages queues are used to serialize message receiving
 const DefaultQueueCount uint = 6
 
 // DefaultMessageQueueSize is the buffer size of each queue mentioned above. (queues are buffered channels)
-const DefaultMessageQueueSize uint = 256
+const DefaultMessageQueueSize uint = 5120
```

Comment:
is this correct still?

Reply:
this number isn't far fetched, indeed can happen. though we still need to tune the exact number

Reply:
consider hare 800 new hare messages happening at the same time, this queue holds all the messages before they are unencrypted so it might have multiple copies of the actual same message (from different neighbors)